src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -871,6 +871,18 @@ public void HandleContextMenu(Key acceleratorKey, ModifierKeys acceleratorModifi
             }
         }
 
+        public static bool ShouldAutoCompleteTextBeEmpty(string queryText, string autoCompleteText)
+        {
+            if (string.IsNullOrEmpty(autoCompleteText))
+            {
+                return false;
```

Comment:
[clarification] shouldn't this be true? ie. the autocompleteText must be empty when the `autoCompleteText` variable is null or empty?

Reply:
The function is to check if autocomplete text should be set to empty. If it is already empty, nothing needs to be done.

Reply:
Sounds good. Thanks for the clarification!